dds/DCPS/GuidUtils.cpp

```diff
@@ -51,6 +52,17 @@ to_string(const GUID_t& guid)
     '.' + to_string(guid.entityId);
 }
 
+bool operator<(const GUID_t& lhs, const GUID_t& rhs)
```

Comment:
@mitza-oci I know historically we haven't overloaded operator< for our set & map types, even though the syntax gets a lot cleaner, but I can't remember if there was a great reason (e.g. does FACE require avoiding operator overload?). Think I would prefer we stick with one approach to minimize duplicate / unused code, which will unfortunately mean getting rid of this function and switching to specify Guid_tKeyLessThan below. I'm OK with leaving both of the template functions, since they won't get instantiated unless they're used.

Reply:
I don't know of any requirement to avoid overloading.
At this point in the release cycle we should be consistent with how it's already done.  We can revisit it and change it everywhere some other time.